flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/source/hybrid/HybridSourceEnumeratorState.java

```diff
@@ -21,18 +21,25 @@
 /** The state of hybrid source enumerator. */
 public class HybridSourceEnumeratorState {
     private final int currentSourceIndex;
-    private final Object wrappedState;
+    private byte[] wrappedStateBytes;
+    private final int wrappedStateSerializerVersion;
```

Comment:
```suggestion
    private final int serializerVersion;
```

Reply:
Considered that as well and prefer the verbose name to make clear that this is the serializer version for the underlying state vs that of `HybridSourceEnumeratorState`.